VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
```

Comment:
Apparently they are planning to move them to `.vs` soon, so, y'know

![image](https://cloud.githubusercontent.com/assets/7574/11604483/f10a1904-9aa1-11e5-8e8f-26a2d8d13139.png)